cmd/extproc/mainlib/main.go

```diff
@@ -70,6 +71,11 @@ func parseAndValidateFlags(args []string) (extProcFlags, error) {
 	)
 	fs.IntVar(&flags.metricsPort, "metricsPort", 1064, "port for the metrics server.")
 	fs.IntVar(&flags.healthPort, "healthPort", 1065, "port for the health check HTTP server.")
+	fs.StringVar(&flags.metricsHeaderNames,
+		"metricsHeaderNames",
```

Comment:
let's add a prefix "request" just in case later we need response header use case
```suggestion
		"metricsRequestHeaderNames",
```